Source/CFString.c

```diff
@@ -44,15 +44,20 @@
 #include <stdio.h>
 #include <string.h>
 
-#if HAVE_UNICODE_UNORM_H
+#if defined(HAVE_UNICODE_UNORM2_H)
+#include <unicode/unorm2.h>
+#elif defined(HAVE_UNICODE_UNORM_H)
```

Comment:
If we're going to start using unorm2, I would like to remove support for the deprecated unorm function, then. UNormalizer2 has been available since version 4.4 and we required version >49. The only reason I used unorm was because this code dates back to version 4.0 of ICU.